packages/core/src/components/forms/_common.scss

```diff
@@ -59,8 +59,8 @@ $control-group-stack: (
 @function input-transition-shadow($color: $input-shadow-color-focus, $focus: false) {
   @if $focus {
     @return
-      border-shadow(1, $color, 1px),
-      border-shadow(0.3, $color, 3px);
+      inset border-shadow(1, $color, 1px),
```

Comment:
add inset focus border for light theme and reduce outer border to 2px
![Screen Shot 2021-12-01 at 16 37 54](https://user-images.githubusercontent.com/5934779/144318097-bb7267ed-8351-4aa3-8b72-3379a6e14112.png)